source/clear-linux/guides/maintenance/swupd-mixer-integration.rst

```diff
@@ -0,0 +1,117 @@
+.. _swupd_mixer_integration:
+
+Adding custom packages and bundles
+##################################
+
+
+
+Prerequisites
+=============
+
+To use the new swupd mixer feature, you'll need a recent image of Clear
+Linux OS for Intel Architecture with the following bundle installed. If you
+don't have it already, you can add it with the :command:`swupd bundle-add`
+command like so::
+
+  # swupd bundle-add mixer
+
+What is it
```

Comment:
Please use descriptive text for headings. The entire document overuses the word "it". My suggestion here would be:
The software updater mixer tool
Remember that level two headings use asterisks as the underline, not equal signs, those are for level three headings.